src/url.js

```diff
@@ -0,0 +1,28 @@
+import isUrl from 'is-url';
```

Comment:
please make sure to always capitalize initialisms (like `URL`) throughout the code (all lowercase is fine too).